setup.py

```diff
@@ -0,0 +1,46 @@
+import sys
+from setuptools import setup, find_packages
+from pedsnetdcc import __version__
+
+if sys.version_info < (2, 7):
```

Comment:
I don't think we've done anything yet which breaks under 2.X and it would be nice if we kept it that way. The actual compatibility will probably be 2.7 and 3.3+ since 3.0 - 3.2 were apparently ugly ducklings that no one likes anymore.